.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
Tad worried about this, a majority of the PRs will be opened by us and this may just get annoyingly verbose.